packages/shared-core/src/filters.ts

```diff
@@ -401,7 +539,7 @@ export const buildQuery = (filter: SearchFilter[]) => {
         }
       }
     } else if (isLogicalSearchOperator(queryOperator)) {
-      // TODO
+      // ignore
```

Comment:
I know this isn't really part of the PR, but what's the plan here long term?

Reply:
Not a clue haha